src/sage/rings/polynomial/pbori/__init__.py

```diff
@@ -39,5 +39,4 @@
 # Import some high-level modelling functionality
 from .blocks import declare_ring
 from .blocks import HigherOrderBlock, AlternatingBlock, Block
-from .gbrefs import load_file
```

Comment:
Can `load_file` be removed without deprecation?

Reply:
It is not in the global namespace. One can certainly not exclude that it is used by some people importing from pbori maybe. Being not tested, it is probably very much broken anyway.